source/frontend/src/features/properties/map/acquisition/AcquisitionView.test.tsx

```diff
@@ -0,0 +1,151 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { Claims } from 'constants/claims';
+import { FileTypes } from 'constants/index';
+import { mockAcquisitionFileResponse } from 'mocks/mockAcquisitionFiles';
+import { mockLookups } from 'mocks/mockLookups';
+import { mockNotesResponse } from 'mocks/mockNoteResponses';
+import React from 'react';
+import { lookupCodesSlice } from 'store/slices/lookupCodes';
+import { prettyFormatDate } from 'utils';
+import { act, render, RenderOptions, userEvent, waitFor } from 'utils/test-utils';
+
+import { SideBarContextProvider } from '../context/sidebarContext';
+import AcquisitionView, { IAcquisitionViewProps } from './AcquisitionView';
+
+const mockAxios = new MockAdapter(axios);
+
+// mock auth library
+jest.mock('@react-keycloak/web');
+
+const onClose = jest.fn();
+const onSave = jest.fn();
+const onCancel = jest.fn();
+const onMenuChange = jest.fn();
+const onSuccess = jest.fn();
+const onCancelConfirm = jest.fn();
+const onUpdateProperties = jest.fn();
+const canRemove = jest.fn();
+const setContainerState = jest.fn();
+
+// Need to mock this library for unit tests
+jest.mock('react-visibility-sensor', () => {
+  return jest.fn().mockImplementation(({ children }) => {
+    if (children instanceof Function) {
+      return children({ isVisible: true });
+    }
+    return children;
+  });
+});
+
+const DEFAULT_PROPS: IAcquisitionViewProps = {
+  onClose,
+  onSave,
+  onCancel,
+  onMenuChange,
+  onSuccess,
+  onCancelConfirm,
+  onUpdateProperties,
+  canRemove,
+  setContainerState,
+  containerState: {
+    acquisitionFile: mockAcquisitionFileResponse(),
+    isEditing: false,
+    selectedMenuIndex: 0,
+    showConfirmModal: false,
+  },
+  formikRef: React.createRef(),
+};
+
+describe('AcquisitionView component', () => {
+  // render component under test
+  const setup = (
+    props: IAcquisitionViewProps = { ...DEFAULT_PROPS },
+    renderOptions: RenderOptions = {},
+  ) => {
+    const utils = render(
+      <SideBarContextProvider
+        file={{
+          ...mockAcquisitionFileResponse(),
+          fileType: FileTypes.Acquisition,
+        }}
+      >
+        <AcquisitionView {...props} />
+      </SideBarContextProvider>,
+      {
+        store: {
+          [lookupCodesSlice.name]: { lookupCodes: mockLookups },
+        },
+        useMockAuthentication: true,
+        claims: renderOptions?.claims ?? [],
+        ...renderOptions,
+      },
+    );
+
+    return {
+      ...utils,
+      getCloseButton: () => utils.getByTitle('close'),
+    };
+  };
+
+  beforeEach(() => {
+    mockAxios.onGet(new RegExp('users/info/*')).reply(200, {});
+    mockAxios.onGet(new RegExp('notes/*')).reply(200, mockNotesResponse());
```

Comment:
ideally we would like to move away from mockAxios for new tests but given this is a refactor I guess it's fine